### ISSUE TITLE
Add customizable mail subject for mailto

### ISSUE THREAD

**Issue by blippercop** [PrivateBin/PrivateBin#928]
The mail option does not have a subject. It would be nice to have a customizable subject.

```<a href="mailto:someone@yoursite.com?subject=themailsubjecthere">Email Us</a>  ```

**Comment by rugk** [comment]
> The mail option does not have a subject. 

Indeed, we could private a default subject like "Your private [self-destroying] note" or so.

> It would be nice to have a customizable subject.

Customizable by whom?
The user already can when the mail client is opened, so do you mean the server admin in the settings?
